### DISCUSSION TITLE
Tab width

### DISCUSSION THREAD

**Discussion by KathrynBassett** [Aris-t2/CustomCSSforFx#562] [General]
The tabs are all the same width. Is there a way to have them automatically shorten based on "title"? For instance, the "TV" tab being much less wide than "Google Advanced Search" and "Ancestry" being a width between those?

**Comment by Aris-t2** [comment]
Please do not start new discussions. Use the one already available: https://github.com/Aris-t2/CustomCSSforFx/discussions/454

This project does not offer a feature like that.
